clients/githubrepo/tarball.go

```diff
@@ -28,7 +28,7 @@ import (
 	"strings"
 	"sync"
 
-	"github.com/google/go-github/v53/github"
+	"github.com/google/go-github/v59/github"
```

Comment:
I updated this because the sbom api stuff wasn't in v53. Looking again, it was added in v54 of the package. I could back it down to that for less of a jump.

Reply:
I backed this out since we're no longer using the sbom api endpoint